src/main/java/org/cascadebot/cascadebot/commands/fun/RandomNumberCommand.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.fun;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import java.io.IOException;
+import java.util.Random;
+
+public class RandomNumberCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        Random randomObj = new Random();
+        if (context.getArgs().length > 0) {
+            String argRaw = context.getArg(0);
+            if (context.isArgInteger(0) == false) {
+                context.getTypedMessaging().replyInfo("Please provide numbers only");
+            }
+
+            else {
+                int argOne = Integer.parseInt(argRaw);
```

Comment:
You should probably use `context.getArgAsInteger(0)` and remove `argRaw`

Reply:
+1 on this

Reply:
On the way